src/ripple.js

```diff
@@ -4,7 +4,7 @@ var Ripple = {
         // Default values.
         var props = {
             event: 'mousedown',
-            transition: 600
+            transition: 950
```

Comment:
it's a default value for a prop. Did you check if we don't set it anywhere in the code?

Reply:
We do not overwrite.